document/src/main/java/com/ritense/document/service/impl/JsonSchemaDocumentDefinitionService.java

```diff
@@ -159,6 +159,25 @@ public Optional<JsonSchemaDocumentDefinition> findLatestByName(String documentDe
         return Optional.ofNullable(definition);
     }
 
+    @Override
+    public Optional<JsonSchemaDocumentDefinition> findByNameAndVersion(String documentDefinitionName, long version) {
+        DocumentDefinition.Id documentDefinitionId = JsonSchemaDocumentDefinitionId.existingId(documentDefinitionName, version);
```

Comment:
If you're going to use `final var` on the next line, please be consistent with it.